### ISSUE TITLE
HIPAA Guides from Stedi Do Not Match TransactionSet from Bootstrap

### ISSUE THREAD

**Issue by russellpekala** [Stedi-Demos/bootstrap#69]
I cloned a HIPAA guide directly from Stedi, and tried to use it in a bootstrap flow.

The "transactionSet" of the cloned guide is "837-Q1". The bootstrap repo has this not matching the transactionSet of just 837, leading to an error in this function (below)
<img width="690" alt="image" src="https://user-images.githubusercontent.com/23023662/221031834-b2458118-506b-4798-90d3-e6037f31a367.png">

**Comment by russellpekala** [comment]
I cannot figure out how to change the guide.target property

**Comment by russellpekala** [comment]
a dumb fix i did locally is this 
<img width="604" alt="image" src="https://user-images.githubusercontent.com/23023662/221045294-ec759735-7d6b-48f8-b641-fb39ac9531c3.png">

**Comment by russellpekala** [comment]
Any update on this? It is very frustrating to hardcode a workaround here. Furthermore, when I try to use the same guide to generate EDI from JSON, it requires that I set transactionSet to "837" instead of "837-Q1".


![image](https://user-images.githubusercontent.com/23023662/225753752-54509c6e-d4ee-423a-b01c-f5a05f1b1ae1.png)

**Comment by russellpekala** [comment]
![image](https://user-images.githubusercontent.com/23023662/225756045-3133b245-f891-4c83-8d37-3a19618244c3.png)

**Comment by russellpekala** [comment]
too lazy to make PR but the above would fix

**Comment by dkanter14** [comment]
As per discussion on Friday 3/24, a fix for this is in and will be merged & deployed Monday 3/27.

**Comment by nddery** [comment]
There was an issue in the HIPAA guides available in the [catalog](https://www.stedi.com/app/guides/view/hipaa), where the guide's `target` had an incorrectly set transaction set ID (per this bug report) - this isn't editable by customers. We've migrated the affected guides and are in the process of reaching out to customers who have one of these guides in their accounts.

Closing this issue as @russellpekala issue should now be fixed. Thank you for reporting the problem!
